main.c

```diff
@@ -1,13 +1,40 @@
 #include <stdio.h>
+#include <stdlib.h>
+
+struct Node {
+    int data;
+    struct Node* next;
+};
 
 int main() {
-    // TODO: build a linked-list version of this array
-    int numbers[] = {1, 2, 3, 4, 5};
 
-    for (int i = 0; i < 5; i++) {
-        printf("%d\n", numbers[i]);
+    struct Node* head = (struct Node*)malloc(sizeof(struct Node));
+    struct Node* second = (struct Node*)malloc(sizeof(struct Node));
+    struct Node* third = (struct Node*)malloc(sizeof(struct Node));
+    struct Node* fourth = (struct Node*)malloc(sizeof(struct Node));
```

Comment:
Consider using a loop to initialize the linked list instead of manually allocating and linking each node.